### PR TITLE
Evaluation effector arm

### PR DESCRIPTION
**Pull request by edwced**

### CODE REVIEW

File: report/finished/Edward_C_evalulation_Effector_Arm.tex [hunk 1]
```diff
@@ -0,0 +1,19 @@
+
+\subsection{Robotic arm}
+In the beginning many different arm choices was discussed. One solution was using a simpler forklift design that just moves up and forward in a rigid motion without rotation.
```
Comment (Partins) [inline report/finished/Edward_C_evalulation_Effector_Arm.tex]: were* discussed

File: report/finished/Edward_C_evalulation_Effector_Arm.tex [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+This solution would suffice. However, we decided to go with a arm design with links connected with motors.
+This is a more complex solution but opens up translation and rotation in 3D-space.
+The design allows the arm to move to any spot in the defined work space.
+to pick up the piece at an angle from factory.
+A limitation is the fact that we decided to keep the of the arm static and parallel to the ground.
```
Comment (Partins) [inline report/finished/Edward_C_evalulation_Effector_Arm.tex]: "...fact that a decision was made..."

File: report/finished/Edward_C_evalulation_Effector_Arm.tex [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+So we can't rotate the end effector. By adding one more motor this can be done. However, we deemed it not necessary for the task at hand.
```
Comment (Partins) [inline report/finished/Edward_C_evalulation_Effector_Arm.tex]: "So the end effector cannot rotate". "By adding one extra motor rotation of the en effector can be achived"

**Review by Partins — COMMENTED** [review]
Good but try to avoid "we"

File: report/finished/Edward_C_evalulation_Effector_Arm.tex [hunk 4]
```diff
@@ -0,0 +1,19 @@
+
+\subsection{Robotic arm}
+In the beginning many different arm choices was discussed. One solution was using a simpler forklift design that just moves up and forward in a rigid motion without rotation.
+This solution would suffice. However, we decided to go with a arm design with links connected with motors.
+This is a more complex solution but opens up translation and rotation in 3D-space.
```
Comment (NiklasDahlquist1) [inline report/finished/Edward_C_evalulation_Effector_Arm.tex]: "opens up **for** translation"  
I don't think it is wrong but maybe it gets clearer?

File: report/finished/Edward_C_evalulation_Effector_Arm.tex (hunk 2 above)
Comment (NiklasDahlquist1) [inline report/finished/Edward_C_evalulation_Effector_Arm.tex]: "keep the **(?)** of the arm"
Missing a word here

File: report/finished/Edward_C_evalulation_Effector_Arm.tex (hunk 3 above)
Comment (NiklasDahlquist1) [reply report/finished/Edward_C_evalulation_Effector_Arm.tex]: "it was deemed that it was not necessary", change the "we"

File: report/finished/Edward_C_evalulation_Effector_Arm.tex [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+The design allows the arm to move to any spot in the defined work space.
+to pick up the piece at an angle from factory.
+A limitation is the fact that we decided to keep the of the arm static and parallel to the ground.
+So we can't rotate the end effector. By adding one more motor this can be done. However, we deemed it not necessary for the task at hand.
+
+\subsection{End effector}
+In order to pick up the object a gripping mechanism was needed, different choices was discussed in the early stages.
+However, we settled for the simple solution of using a claw.
```
Comment (NiklasDahlquist1) [inline report/finished/Edward_C_evalulation_Effector_Arm.tex]: Same as earlier, avoid "we"

File: report/finished/Edward_C_evalulation_Effector_Arm.tex [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+By rotating the shaft of motor the claw either retracts or expands.
```
Comment (NiklasDahlquist1) [inline report/finished/Edward_C_evalulation_Effector_Arm.tex]: "of **the** motor" 
Add what motor we are talking about, for example "By rotating the shaft of the motor, connected to the claw, the claw either retracts..."

**Comment by edwced** [comment]
Fixed minor issues, like "we" and input. Done from my part?

**Pull request merged**